lib/smart_proxy_salt/cli.rb

```diff
@@ -62,6 +65,57 @@ def autosign_list
           end.map(&:chomp)
         end
 
+        def append_value_to_file(filepath, value)
+          begin
+            raise "No such file: #{filepath}" unless File.exist?(filepath)
+
+            file = open(filepath, File::RDWR)
+            found = false
+            file.each_line { |line| found = true if line.chomp == value }
+            file.puts value unless found
+            file.close
+
+            logger.info "Added an entry to '#{filepath}' successfully."
+            result = true
+          rescue SystemCallError => e
+            logger.info "Attempted to add an entry to '#{filepath}', but an exception occurred: #{e}"
+            result = false
```

Comment:
I would move the "result=false" to line 68 just before the begin block.

Reply:
Implemented as suggested. Did the same for `append_value_to_file`.